lib/engine/game/g_1848/game.rb

```diff
@@ -369,6 +387,15 @@ def new_auction_round
           ])
         end
 
+        def stock_round
+          Engine::Round::Stock.new(self, [
+            Engine::Step::DiscardTrain,
```

Comment:
Is there a situation in 1848 where a corp could or would need to discard a train during the stock round?

Reply:
Found it in the rules.

Reply:
where in the rules? I just copied the super method and modified one step, didn't want to touch the others in case they are relevent